experimental/basisLieHighestWeight/MB.jl

```diff
@@ -0,0 +1,364 @@
+module MB
```

Comment:
Why `MB`?

Reply:
MB stand for MonomialBasis, but I changed it to BasisLieHighestWeight now.